modules/pytket-cirq/pytket/extensions/cirq/backends/cirq_utils.py

```diff
@@ -33,24 +34,41 @@ def _get_default_uids(
             cirq_circuit.all_qubits()
         )
 
-        cirq_measures = [c[1] for c in cirq_circuit.findall_operations(is_measurement)]
+        ordered_tket_qubits = []
+        for cirq_qubit in ordered_cirq_qubits:
+            if isinstance(cirq_qubit, NamedQubit):
+                ordered_tket_qubits.extend(
+                    [qb for qb in tket_circuit.qubits if qb.reg_name == cirq_qubit.name]
```

Comment:
If `NamedQubit` objects don't have indexing, I assume this works because the TKET circuit has many different registers all slightly named due to a different index?